apps/patient/src/views/Status.tsx

```diff
@@ -35,13 +35,18 @@ export const Status = () => {
   const [searchParams] = useSearchParams()
   const orderId = searchParams.get('orderId')
   const token = searchParams.get('token')
+  const courier = searchParams.get('courier')
```

Comment:
Do we need this if we have the check for the pharmacy id?

Reply:
There's no pharmacy details on the order if we immediately transition to the status view...

<img width="206" alt="image" src="https://user-images.githubusercontent.com/3934326/230632963-feec3b32-3ee3-4f95-84b7-474d5b735f93.png">